### DISCUSSION TITLE
Can I change the path of querylog.json?

### DISCUSSION THREAD

**Discussion by Jefffish09** [AdguardTeam/AdGuardHome#4467] [Q&A] [answered]
Due to the large size of querylog.json, I need to move it to other location which has larger space.

But in the Configuration wiki, I cannot find any item to modify the path of querylog.json, so can I change this path currently?

![image](https://user-images.githubusercontent.com/44800122/162169255-6595b48b-d8cd-4bfa-a09b-565d1c3f4749.png)

**Comment by ainar-g** [comment, marked as answer]
Unfortunately, this cannot be done currently.  You can only change the whole working directory for AdGuard Home.  We plan to address this in v0.108.

**Comment by t3dium** [comment]
for the time being, I’d suggest to symlink ~~/tmp/adguardhome/~~ /opt/AdGuardHome/data to a custom path, until the new update is pushed out

**Reply by Jefffish09** [reply]
I'm new to Linux. Could you pls tell more details how to 'symlink /tmp/adguardhome/ to a custom path'?

**Reply by t3dium** [reply]
> I'm new to Linux. Could you pls tell more details how to 'symlink /tmp/adguardhome/ to a custom path'?


A symlink just points the program accessing the path, to the path you want.
`ln -s /from/ /path/to/destination`

Also, I was wrong with the path in my prev comment. It’s actually stored in /opt/AdGuardHome/data/. So the command would be:
`sudo ln -s /opt/AdGuardHome/data/ /path/to/destination`

You’d want to stop adguard home (`sudo /opt/AdGuardHome —service stop`) copy the folder to the destination and delete the old data folder **beforehand** tho. (cp -r with paths, and then rm -rf path)
